add-ons/management-infra/README.md

```diff
@@ -0,0 +1,20 @@
+# management-infra Add-on
+An addon to create objects to support ManageIQ or CloudForms 
+
+NOTE: Requires Origin >= 3.6.0
+
+Verify you have installed these addons, by following the [general readme](../../Readme.adoc#download-and-use-community-add-ons).
```

Comment:
Not following this sentence...

Reply:
Copied from prometheus. I guess it is for general info about addons. Tell me if you prefer to delete it.

Reply:
I am confused what it means; which 'these' is meant ;-). I guess it is not helpful information here.

I guess it would be better if you also include a PR for the other addons, so you can link to the folders or relevant README.md of these addons. I guess this is what is meant here?